src/app/settings/vault-timeout-input.component.ts

```diff
@@ -19,4 +19,9 @@ import { VaultTimeoutInputComponent as VaultTimeoutInputComponentBase } from "js
     },
   ],
 })
-export class VaultTimeoutInputComponent extends VaultTimeoutInputComponentBase {}
+export class VaultTimeoutInputComponent extends VaultTimeoutInputComponentBase {
+  //checks if input minutes is valid
+  getValidMinutesInput(): boolean {
```

Comment:
This method repeats the validation check you did in your jslib PR. Suggested solution here: https://github.com/bitwarden/jslib/pull/802#discussion_r882324921